chartfx-chart/src/main/java/de/gsi/chart/renderer/spi/ContourDataSetRenderer.java

```diff
@@ -722,8 +516,7 @@ private static double quantize(final double value, final int nLevels) {
         // original: return Math.round(value * nLevels) / (double) nLevels;
     }
 
-    private static void sobelOperator(final double[][] input, final double[][] output, final double zMin,
-            final double zMax, final double level) {
+    private static void sobelOperator(final double[][] input, final double[][] output, final double zMin, final double zMax, final double level) {
```

Comment:
Codacy found an issue: [The method 'sobelOperator(double, double, double, double, double)' has an NPath complexity of 262](https://app.codacy.com/manual/GSI/chart-fx/pullRequest?prid=5780327)